test.py

```diff
@@ -1253,8 +1254,8 @@ def parse_cmd_line() -> argparse.Namespace:
                         help="Save test log output on success.")
     parser.add_argument('--list', dest="list_tests", action="store_true", default=False,
                         help="Print list of tests instead of executing them")
-    parser.add_argument('--skip', default="",
-                        dest="skip_pattern", action="store",
+    parser.add_argument('--skip',
+                        dest="skip_patterns", action="append",
```

Comment:
i like this option. but instead of an ignore list, what about a include list, so that only the tests matched by the pattern specified by the "include list" are added? sometime, when i spot a failing test, i feel tempted to reproduce it locally, i always have to comment out the other tests in that .py file, and rerun it with `test.py` if that very test cannot be launched using `pytest` directly, or it's complicated to do so.

Reply:
@tchaikov This '--skip' argument only matches the test file names and not the individual names inside them. 
But what you ask for is doable by passing an expression to `-k` argument of pytest.
I have implemented a version that takes in test patterns and converts them into an expression understood by pytest - https://github.com/scylladb/scylladb/pull/17223.
Let me know your comments on that.